### PR TITLE
refactor: neuron-wallet jest environment

### PR DESCRIPTION
**Pull request by BaojunCZ**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
   },
   "devDependencies": {
     "@cryptape/sdk-ts-config": "0.0.1",
-    "@types/jest": "^24.0.0",
+    "@types/jest": "^23.0.0",
```
Comment (Keith-CY) [inline package.json]: If the neuron-ui uses jest@23 and neuron-wallet uses jest@24, the @types/jest is better to be added in them, respectively.

File: packages/neuron-wallet/package.json [hunk 2]
```diff
@@ -23,19 +23,38 @@
...
+        "moduleFileExtensions": [
+          "ts",
+          "js",
+          "json",
+          "node"
+        ],
+        "setupTestFrameworkScriptFile": "<rootDir>/setupTests.ts"
+      },
+      {
+        "displayName": "node",
```
Comment (Keith-CY) [inline packages/neuron-wallet/package.json]: Why does the neuron wallet have two projects

File: package.json (hunk 1 above)
Comment (BaojunCZ) [reply package.json]: They use jest@23 together now.

File: packages/neuron-wallet/package.json (hunk 2 above)
Comment (BaojunCZ) [reply packages/neuron-wallet/package.json]: “electron” project need electron environment."node" project doesn`t

Comment (Keith-CY) [reply packages/neuron-wallet/package.json]: But they cover the same test files

Comment (BaojunCZ) [reply packages/neuron-wallet/package.json]: different Regex."electron" is *.test.electron.ts,"node" is “.test.ts”

**Pull request merged**